### PR TITLE
basic site splash page

### PR DESCRIPTION
**Pull request by jpalmieri**
This is meant to:
* reproduce what we already have on kepler452b.org (via copypasta)
* clean up some of the html and css
* make it a little bit more mobile friendly/responsive (one css media viewport query)

Next steps:
* put this on a server, set up a simple deploy process (Eric?)
  * Heroku would be simplest and it seems like we'd only have to do the $25/month option
  * I'm ok with other options though
  * should we have a staging server too? maybe unnecessary, but would be easy with heroku
  * stefanie has the current site on bluehost, but I'm not sure how we'd do that. We'd at least need her bluehost login info (not just the FTP creds, I assume), and idk about the pricing
* configure DNS to have kepler452b.org point to the new server (Eric?). I think we'll probably need stefanie's bluehost login for this
* work on parallax effect for splash page (joe)
  * current site splash page will 'slide up' via parallax effect to reveal list of performers/collaborators/djs
  * this list will be replaced by 'constellations' in a later iteration, once mocks are ready (stefanie is working on mocks now)

### CODE REVIEW

File: app/assets/stylesheets/welcome.scss [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+  height: 100%;
+	background-color: #333333;
+	background-image: image-url('kepler-bg-sm.gif');
+	background-size: cover;
+	font-family: 'Roboto', Helvetica, Verdana, Arial, sans-serif;
+}
+
+a:link {
+  text-decoration:none;
+	color: #666666;
```
Comment (sfjuggernaut) [inline app/assets/stylesheets/welcome.scss]: https://www.youtube.com/watch?v=hv_zJrO_ptk

File: app/assets/stylesheets/welcome.scss [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+  font-weight:100;
+  text-align:center;
+  font-size: 48px;
+  letter-spacing:5px;
+  padding: 0;
+}
+
+h2 {
+  font-style:normal;
+	font-weight:300;
```
Comment (sfjuggernaut) [inline app/assets/stylesheets/welcome.scss]: Not a big deal, any idea what's up with the weird indentation spacing that's showing up?

File: app/views/layouts/application.html.erb [hunk 3]
```diff
@@ -1,10 +1,16 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>Kepler</title>
+    <title>kepler 452b</title>
     <%= csrf_meta_tags %>
+    <meta name="description" content="kepler 452b Berlin Party Collective">
```
Comment (sfjuggernaut) [inline app/views/layouts/application.html.erb:6]: https://www.youtube.com/watch?v=WccfbPQNMbg

**Comment by sfjuggernaut** [comment]
Minor nit with regards to indentation white space, but otherwise looks good !

**Comment by sfjuggernaut** [comment]
I'm happy to figure out deploy process and DNS.

File: app/assets/stylesheets/welcome.scss (hunk 1 above)
Comment (jpalmieri) [reply app/assets/stylesheets/welcome.scss]: 😈

File: app/assets/stylesheets/welcome.scss (hunk 2 above)
Comment (jpalmieri) [reply app/assets/stylesheets/welcome.scss]: shit, I copy pasted from the site. Some of them must be tabs.

File: app/views/layouts/application.html.erb (hunk 3 above)
Comment (jpalmieri) [reply app/views/layouts/application.html.erb:6]: image

**Comment by jpalmieri** [comment]
whoops...accidentally added in an extra file I was working on in a different branch (app/assets/javascripts/cover-slide.js). I'll remove as a commit to master

**Pull request merged**